packages/xstate-react/test/useActor.test.tsx

```diff
@@ -65,7 +63,8 @@ describe('useActor', () => {
     );
   });
 
-  it('invoked actor should be able to receive (deferred) events that it replays when active', (done) => {
+  // TODO: how deferred actors are currently handled is quite hacky, we should preserve actor identity for deferred actors
+  it.skip('invoked actor should be able to receive (deferred) events that it replays when active', (done) => {
```

Comment:
oops, I forgot to recheck this - I'll do that later today, but this is yet again a problem caused by deferred children :S

Reply:
Somewhat weirdly those tests now "just pass". In a way - that's good, in another way - I'm not yet 100% sure if this is not a false positive. I need to recheck this.